java/src/main/java/scenarios/crewcreation/CharacterCreation.java

```diff
@@ -36,10 +36,25 @@ public static Captain runCharacterCreation() {
 
         List<Integer> unassignedAbilityScores = DiceRoller.rollAbilityScores();
 
-        int meanScore = (int) unassignedAbilityScores.stream().mapToInt(i -> i).average().orElse(0);
+        printMeanScoreFlavorText((int) unassignedAbilityScores.stream().mapToInt(i -> i).average().orElse(0));
```

Comment:
This needs to go below the next printLn, because we should be showing what the scores are before giving the flavor text.